src/components/Layout/index.js

```diff
@@ -12,7 +12,7 @@ class Layout extends React.Component {
   componentWillMount () {
     WebFont.load({
       google: {
-        families: ['El Messiri:300,400,700', 'Raleway:300,400,400i,700,700i']
+        families: ['El Messiri:300,400,700', 'Raleway:300,400,400i,600,700,700i,800']
```

Comment:
do we need the Raleway font in 800 font-weight? Fonts are expensive (in terms of size), we shouldn't have to have too many loaded.

Reply:
Yeah you're right. In the end i didn't use the 800 in my solution. I'll correct that. In case it's a big hitch to load 600, we could experiment with changing only the font-size.